.github/workflows/jdbc-tests-with-non-default-server-collation.yml

```diff
@@ -0,0 +1,111 @@
+name: JDBC Tests with Non Default Server Collation
```

Comment:
I have added logical replication support with JDBC framework, let's include those changes in this as well. Refer https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/214ea5831d3077bb900ac1753609364a3edae4c9

Reply:
Done